src/test.js

```diff
@@ -64,6 +64,10 @@ export default function Test( settings ) {
 		this.async = false;
 		this.expected = 0;
 	} else {
+		this.module.unskippedTests.push( {
```

Comment:
I actually considered keeping just one array of tests (`module.tests`) and adding a `skip` flag to its entries. That way we wouldn't need anything extra because we can always find out if the current test is the first/last unskipped test by looking through `module.tests` (and child modules) and comparing `testId` to `this.testId` and looking at `skip` flags.

I didn't go in that direction because `module.tests` is passed into various callbacks as the context, and I didn't want to make any public-facing changes. If you prefer that route (then we don't need any new counts or arrays or anything, just a `skip` flag on the `module.tests` entries), I can certainly do that.

If not, can you elaborate on exactly where you want the `skipped` flag? From your comment I'm not entirely sure.